psignifit/psigniplot.py

```diff
@@ -678,7 +678,7 @@ def plot_posterior_samples(
             gamma=params_samples['gamma'][idx],
             lambd=params_samples['lambda'][idx],
         )
-        ax.plot(x, y, alpha=samples_alpha, color=samples_color)
+        ax.plot(x, y, alpha=samples_alpha, color=samples_color, zorder=-1)
```

Comment:
As far as I understand, a negative z-order would place the samples behind the plotting elements of the axis, which might make them disappear if the user sets a non-transparent background for the plot. I think a value of 0.5 might be better (e.g. https://matplotlib.org/stable/gallery/misc/zorder_demo.html#sphx-glr-gallery-misc-zorder-demo-py)